gateway-ha/src/main/resources/gateway-ha-persistence.sql

```diff
@@ -75,4 +75,14 @@ CREATE TABLE IF NOT EXISTS exact_match_source_selectors (
 
     PRIMARY KEY (environment, source, query_type),
     UNIQUE (source, environment, query_type, resource_group_id)
-);
\ No newline at end of file
+);
+
```

Comment:
Just curious if/how these database entries get managed. Any chance these could build up over time and cause an issue?